translations/config.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+type ConfigType = {
+  sourceLanguage: string,
+  supportedLanguages: { [languageCode: string]: string },
```

Comment:
I think this type is not correct, is it? Shouldn't the type be `{ [languageCode: string]: {| rtl?: boolean |} }`? Or depending on my following suggestion: `{ [languageCode: string]: {| rlt: boolean |} }`